refresh-secrets.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+vault token revoke -self
+unset VAULT_TOKEN
+source /infrastructure/import-secrets.sh
```

Comment:
This should be a relative path. I often use the container with `/src` mounted.

Reply:
From my understanding `/infrastructure` should always be present. Relative to what? Current working directory could be any. The source will not find the file then. What am I missing?

Reply:
During the development I use `/src` mount for example, if this is kept as is, I need to rebuild the container each time I change the `import-secrets.sh` to test the `refresh-secrets.sh`

Reply:
Fixed.